templates/user/dashboard/repolist.tmpl

```diff
@@ -46,6 +46,8 @@ data.organizations = [{{range .Orgs}}{'name': {{.Name}}, 'num_repos': {{.NumRepo
 data.isOrganization = false;
 data.organizationsTotalCount = {{.UserOrgsCount}}
 data.canCreateOrganization = {{.SignedUser.CanCreateOrganization}}
+{{else}}
+data.organizationParam = '?org=' + {{.ContextUser.ID}};
```

Comment:
Could we pass "organizationId" instead of a URL param? The ID seems more general.

Reply:
Ah, I found the way to use `if` inside the string.
Thanks for your review.